R/update.R

```diff
@@ -91,8 +91,8 @@ ci_update <- function(profile = 'lesson-requirements', update = 'true', repos =
     if (length(header)) {
       header <- seq(min(header) - 1L)
     }
-    footer <- seq(which(startsWith(trimws(update_report), "- Lockfile")),
-      length(update_report))
+    footer <- tryCatch(seq(which(startsWith(trimws(update_report), "- Lockfile")),
+      length(update_report)), error = function(e) length(update_report))
```

Comment:
For context: this line was failing if `update_report` did not have a line that starts with `- Lockfile`, so the `tryCatch()` provides a fallback to getting the final position of the file. It's a workaround, but it will mean that the updates will not randomly fail.

Reply:
Thank you for taking the time to make the required changes!